contactform/ContactFormPlugin.php

```diff
@@ -79,27 +79,70 @@ public function getSettingsHtml()
 	 */
 	public function setSettings($values)
 	{
-		if (!$values)
+    $settings = $this->prepSettings($values);
+
+    // Craft::dd(craft()->request->getPost('settings'));
+
+		parent::setSettings($settings);
+	}
+
+  /**
+	 * @return array
+	 */
+  public function prepSettings($values) {
+
+    if (!$values)
 		{
 			$values = array();
 		}
 
 		if (is_array($values))
 		{
+      $postedSettings = craft()->request->getPost('settings');
```

Comment:
This was the only way I was able to get the entered settings into the database. Each time I submitted the form with the settings in the table field the database would be empty. The order here is...
1. Take the posted value.
2. If the value is not set, take the default value.
3. Finally the config value takes precedence over all.

Hope this helps :)